ep043/README.md

```diff
@@ -0,0 +1,35 @@
+# [introduction to tox (beginner - intermediate)](https://youtu.be/75WBE_qbpGk)
+
+Today I explain the basics of using `tox` as an automation / testing tool!
+
+## Setup commands
+
+```bash
+git clone git@github.com:asottile/pyupgrade.git
```

Comment:
I don't use `.git` on these -- it's not necessary

Reply:
Do you want me to remove it? I just copied what GitHub gives me by default from the `Add Code` button.

Reply:
yes please!